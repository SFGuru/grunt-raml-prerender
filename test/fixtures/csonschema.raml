#%RAML 0.8
# example file for testing csonschema

title: csonschema RAML API
baseUri: https://{server.com}/api
version: 1.2
/resource:
  displayName: a resource can have a display name
  description: |
    we like bold descriptions
    --------------------
    a resource can have a __description__ too?
  post:
    description: |
      attention
      =========

      we can post to this API too
    body:
      application/json:
        schema: |
          {
            "$schema": "http://json-schema.org/draft-03/schema",
            "description": "a long description <b>here we go bold</b> and there",
            "properties": {
                "input": {
                    "required": false,
                    "type": "string"
                }
            },
            "required": false,
            "type": "object"
          }
        example: |
          {
            "input": "stuff"
          }
    responses:
      200:
        description: |
          A long description goes here.
        body:
          application/json:
            schema: |
              {
                "$schema": "http://json-schema.org/draft-03/schema",
                "id": "simple",
                "properties": {
                    "input": {
                        "required": false,
                        "type": "string"
                    }
                },
                "required": false,
                "type": "object"
              }
            example: |
              {
                "input": "stuff"
              }