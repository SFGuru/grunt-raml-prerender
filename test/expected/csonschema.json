{"title":"csonschema RAML API","baseUri":"https://{server.com}/api","version":"1.2","protocols":["HTTPS"],"resources":[{"displayName":"a resource can have a display name","description":"<h2 id=\"welikebolddescriptions\">we like bold descriptions</h2>\n\n<p>a resource can have a <strong>description</strong> too?</p>","relativeUri":"/resource","methods":[{"description":"<h1 id=\"attention\">attention</h1>\n\n<p>we can post to this API too</p>","body":{"application/json":{"schema":"{\n  \"$schema\": \"http://json-schema.org/draft-03/schema\",\n  \"properties\": {\n    \"input\": {\n      \"required\": false,\n      \"type\": \"string\"\n    }\n  },\n  \"required\": false,\n  \"type\": \"object\"\n}","example":"{\n  \"input\": \"stuff\"\n}"}},"responses":{"200":{"description":"<p>A long description goes here.</p>","body":{"application/json":{"schema":"{\n  \"$schema\": \"http://json-schema.org/draft-03/schema\",\n  \"id\": \"simple\",\n  \"properties\": {\n    \"input\": {\n      \"required\": false,\n      \"type\": \"string\"\n    }\n  },\n  \"required\": false,\n  \"type\": \"object\"\n}","example":"{\n  \"input\": \"stuff\"\n}"}}}},"protocols":["HTTPS"],"method":"post","bodyDescription":"a long description <b>here we go bold</b> and there"}]}],"baseUriParameters":{"server.com":{"type":"string","required":true,"displayName":"server.com"}}}